01-shopping-cart-service-scala/deployment/Akka-CRD-defaults.yml

```diff
@@ -0,0 +1,93 @@
+apiVersion: apps/v1
```

Comment:
This YAML shows a complete manifest with default values. I think the user YAML should be overlaid on top.